src/pages/components/form/usage.mdx

```diff
@@ -117,215 +115,378 @@ do you gain by collecting this information?
 
 ## Formatting
 
-All forms are comprised of six elements:
-
-- **Labels:** Inform users what the corresponding input fields mean.
-- **Input fields:** Enable users to provide information. Information can be
-  entered through a variety of different input fields ranging from text fields,
-  checkboxes, and many other types.
-- **Helper text:** Provides assistance on how to fill out a field. Helper text
-  is optional.
-- **Placeholder text:** Hints at what goes into a field. Placeholder text is
-  optional.
-- **Actions:** Allow users to submit a form.
-- **Validation:** Ensures the data submitted by the user conforms to acceptable
-  parameters.
+### Anatomy
 
 <Row>
 <Column colLg={8}>
 
-![effective form design](images/form-usage-1.png)
+<Tabs>
 
-</Column>
-</Row>
+<Tab label="Fixed">
 
-### Button placement
+![Anatomy of a fixed form](images/form-usage-1.png)
```

Comment:
Anatomy image (fixed):
This abbreviation should be `etc.` not ect.
<img width="635" alt="Screen Shot 2023-01-24 at 12 44 54 PM" src="https://user-images.githubusercontent.com/43969356/214387590-3483733b-cdb4-4f2c-9496-03ada688817b.png">